cli/root.go

```diff
@@ -154,6 +175,8 @@ func Execute() {
 	configureConfigCmd(app)
 	configureTemplateCmd(app)
 
+	app.Action(initMatchersCompat)
```

Comment:
This is more or less documented in the doc comments for `app.Action`:

> Action callback to call when all values are populated and parsing is complete, but before any command, flag or argument actions.

If you still want it to be documented here I'm happy to make that change, but just wanted to share this information in case it changes your opinion.